website/docs/tasks.md

```diff
@@ -236,9 +241,10 @@ This is a convenience wrapper for
 
 ### `helpers.withPgClient()`
 
-`withPgClient` gets a `pgClient` from the pool, calls
-`await callback(pgClient)`, and finally releases the client and returns the
-result of `callback`. This workflow can make testing your tasks easier.
+`withPgClient` gets a `pgClient` from the pool that Graphile Worker uses. It
+calls `await callback(pgClient)`, and finally releases the client and returns
+the result of `callback`. This workflow can make testing your tasks easier by
+making it easier to mock `pgClient`.
```

Comment:
One thing to know about me is I almost never use mocks. How it makes testing easier for me is I always return the same client, which can be in a transaction, so I can roll back the changes later; however I’d be careful about how that’s expressed here because I wouldn’t want a user thinking it will always return the same client normally.

Reply:
Gotcha. I think I'll revert this addition for now. When I first read that it would make testing easier, my next question was "why?". Was trying to avoid that, but I think this level of nuance is probably better communicated in a future section on testing, or something like that